### PR TITLE
[9.0][MIG]ldap_populate 

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

File: users_ldap_populate/view/users_ldap.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
             <field name="model">res.company</field>
             <field name="inherit_id" ref="auth_ldap.company_form_view"/>
             <field name="arch" type="xml">
-                <xpath expr="//form[@string='LDAP Configuration']" position="inside">
+                <xpath name="populate_ldap" expr="//field[@name='ldaps']/form" position="inside">
```
Comment (dreispt) [inline users_ldap_populate/view/users_ldap.xml:9]: I don't think you should use `name` in an xpath element.

Comment (gurneyalex) [inline users_ldap_populate/view/users_ldap.xml:9]: why? the name is not translatable (while the string is, which is way the original version was changed I believe)

**Comment by gurneyalex** [comment]
:+1:

File: users_ldap_populate/view/users_ldap.xml (hunk 1 above)
Comment (dreispt) [inline users_ldap_populate/view/users_ldap.xml:9]: My bad, `name` is perfectly OK.
I was just checking if you guys were paying attention :wink:

**Comment by dreispt** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Comment by pedrobaeza** [comment]
You haven't checked that there's a README.rst file and the description is there with the proper OCA's template. Please make another PR with these changes.

**Pull request merged**